### ISSUE TITLE
Question regarding image shape for perceptual loss 

### ISSUE THREAD

**Issue by seungjunlee96** [rosinality/stylegan2-pytorch#94]
Hi,

I wonder why we should reshape image to 256 x 256 ,
even though the default image shape for VGG is 224 x 224.

Thanks!

**Comment by rosinality** [comment]
This model doesn't use imagenet pretrained vgg, so it doesn't need to be restricted on 224px resolutions. (By the way, vgg also uses 256px for evaluation.)

**Comment by seungjunlee96** [comment]
Thanks for your kind reply!
